docs/pages/layout/index.vue

```diff
@@ -0,0 +1,155 @@
+<template>
+
+  <DocsPageTemplate>
+
+    <DocsPageSection title="Overview" anchor="#overview">
+      <p>The goal of good layout is legibility - to aid the user in quickly identifying what they need on the page. This is achieved through effective use of white space, graphics, typography, and controlled use of lines.</p>
+    </DocsPageSection>
+
+    <DocsPageSection title="Spacing" anchor="#spacing">
+      <p>To aid with alignment, all UI elements should be spaced out in increments of 8px. On smaller screens or in more compact spaces, the increment can be reduced to 4px.</p>
+      <p>All margin and padding measurements should be incremented by these units.</p>
+    </DocsPageSection>
+
+    <DocsPageSection title="Containers" anchor="#containers">
+      <p>A container is a shape used to represent an enclosed area. Containers can hold various UI elements such as an image, icon, or surface. They can be flexible and accommodate the width or height of the content within, or they can be rigid and fixed in size.</p>
+      <p>To reduce visual noise, avoid nesting containers in containers.</p>
+      <img src="./containers.png">
+      <h3>
+        Specs
+      </h3>
+      <ul>
+        <li>Background color: tokens.surface (#ffffff)</li>
+        <li>Default border radius: 4px</li>
+        <li>Elevation: 1dp</li>
+        <li>Spacing between containers: 24px</li>
+        <li>Padding: 16px (small screens), 24px (medium and large screens)</li>
+      </ul>
+      <h3>
+        Max width
+      </h3>
+      <p>
+        If a page container contains text, set its maximum width to 1000px in order to aid readability on larger screens.
+      </p>
+      <h3>
+        Dividers
+      </h3>
+      <p>
+        Use dividers to create visual distinction between sections within a container. A divider should span the entire width of a container.
+      </p>
+      <p>
+        Use sparingly and reserve them for situations where white space does not properly separate elements or groups of elements, such as a table, compact list items, or list sections.
+      </p>
+      <img src="./dividers.png">
+      <h3>
+        Specs
+      </h3>
+      <ul>
+        <li>Width: 1px</li>
+        <li>Color: tokens.fineline (#dedede)</li>
+      </ul>
+    </DocsPageSection>
+
+    <DocsPageSection title="Responsive Grids" anchor="#grids">
```

Comment:
Assuming we're using our own writing guidelines, headers should be `Sentence case`

```suggestion
    <DocsPageSection title="Responsive grids" anchor="#grids">
```